keel-slack/src/main/kotlin/com/netflix/spinnaker/keel/slack/callbacks/ManualJudgmentCallbackHandler.kt

```diff
@@ -63,13 +64,25 @@ class ManualJudgmentCallbackHandler(
    * For example, if user gyardeni approved the notification, this function will add:
    * "@Gal Yardeni hit approve on 2021-02-12 11:05:57 AM" and will marked the original text with strikethrough.
    */
-  fun updateManualJudgementNotification(response: BlockActionPayload): List<LayoutBlock> {
+  fun oldUpdateManualJudgementNotification(response: BlockActionPayload): List<LayoutBlock> {
     try {
       val originalCommitText = response.message.blocks[1].getText
-      val originalGitInfoText = response.message.blocks[2].getText
-      val originalUrl = response.message.blocks[2].getUrl
       val action = actionsMap[response.actions.first().value]
 
+      // since we sometimes add a "show more" button, the position of this information changes.
+      //  I cannot figure out (yet) how to scrape the original section and just re-add it to the notification
```

Comment:
Aren't the blocks all exactly the same except for the one added at the bottom to say "X person approved/rejected"? If that's the case, couldn't we just use the original array of blocks and tack that at the end? Why do we need to worry about these positions to begin with?

Reply:
We update the first block to have a different image.

I have been struggling with how to just grab the old blocks but actually, you just gave me an idea!